va/va.h

```diff
@@ -1165,6 +1169,26 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** \brief Attribute value for VAConfigAttribLookAhead*/
+typedef union _VAConfigAttribValEncLookAhead {
+    struct {
+        /** \brief Number of frames to lookahead. Range is [0~100]. 0 means lookahead disabled.*/
+        uint32_t look_ahead_depth               : 8;
+        /** \brief indicate whether the current encoding is in the Look Ahead pass
+         *  if look_ahead_phase == 0, the current encoding is in the actual encoding pass
+         *  if look_ahead_phase == 1, the current encoding is in the Look Ahead pass.*/
+        uint32_t look_ahead_phase               : 1;
+        /** \brief A flag indicates whether Look Ahead Analysis is supported
+         *  if lookahead analysis is supported, application can ignore encoded bitstream.*/
+        uint32_t look_ahead_analysis_support    : 1;
+        /** \brief A flag indicates whether Look Ahead BRC is supported.*/
```

Comment:
what is look ahead BRC?

Reply:
Updated commit has combined these two caps as one as look_ahead_support.